content/articles/how-to-build-a-web-crawler-with-java/index.md

```diff
@@ -0,0 +1,178 @@
+[According to seedscientific.com](https://seedscientific.com/how-much-data-is-created-every-day), we generate 2.5 quintillion bytes of data every day. A significant part of this data is generated through our interactions with the internet. Big organizations worldwide extract and analyze these data for business research purposes to further grow their businesses while maximizing profits.
+
+How do these organizations traverse the web to explore these existing data for their desired purposes? That’s where web crawlers come in. A web crawler is [one of the tools](https://hevodata.com/learn/8-best-web-scraping-tools/) for traversing the internet for data-gathering. It is an automated tool, usually a script written in any suitable programming language, that navigates the web through a series of web pages contained in one another. It is sometimes used interchangeably with Web scraping, a tool that does the actual job of pulling the data from web pages.
```

Comment:
Adding 1 - 2 lines of explanation about web scraping would help. You can add reference links as well.